app/jobs/consume_assessments_job.rb

```diff
@@ -26,46 +26,46 @@ def perform
         if message['response_status'] == 'no_answer_voice'
           # If nobody answered, nil out the last_reminder_sent field so the system will try calling again
           patient.update(last_assessment_reminder_sent: nil)
-          history = History.new
-          history.created_by = 'Sara Alert System'
-          comment = "Sara Alert called this monitoree's primary telephone number #{patient.primary_telephone} and nobody answered the phone."
-          history.comment = comment
-          history.patient = patient
-          history.history_type = 'Contact Attempt'
-          history.save
+          History.contact_attempt(patient: patient, comment: "Sara Alert called this monitoree's primary telephone \
+                                                              number #{patient.primary_telephone} and nobody answered the phone.")
+          unless patient.dependents.blank?
+            create_contact_attempt_history_for_dependents(patient.dependents, "Sara Alert called this monitoree's head \
```

Comment:
I think for all of these calls, it needs to be `patient.dependents.where.not(id: patient.id)` because right the dependents of a patient include itself, so the head of the household gets two history items.

Reply:
Fixed. Made ticket to address this more widely:  https://jira.mitre.org/browse/SARAALERT-590
Didn't want to include in this PR as that would add more to test.